### ISSUE TITLE
Set Accessory Status?

### ISSUE THREAD

**Issue by danielsen2009** [cflurin/homebridge-mqtt#18]
If I reply to getAccessory even with "false" i.e. outlet status it still show as on. Same with contact sensors and switches. How do I correctly respond to the get queries? Or should I only respond if its "On", but then how do I set a false status if the control happens outside of homekit? I'm replying on "homebridge/to/set" with the current values. I just can't seem to get it to work.

**Comment by cflurin** [comment]
let's look at the sample `switch`
```sh
topic: homebridge/from/get
payload: {"name": "flex_lamp", "characteristic": "On"}
```

When Homebridge-mqtt get an `Accessory.get`request from HomeKit it will immediately return the cached value to HomeKit and sends `homebridge/from/get` to the mqtt-broker. You can ignore it or  publish the actual value using `homebridge/to/set`.

**Comment by danielsen2009** [comment]
I'm still having issues with this, if I reply to the call at all it sets the status of "On" to 
"true".. Even if I sent "false" to "homebridge/to/set".

**Comment by cflurin** [comment]
This works for me:

![callback](https://cloud.githubusercontent.com/assets/5056710/21015036/a3ee8b36-bd60-11e6-84cc-ee98fbda35e8.png)

```sh
[{"id":"8e610acf.bf82f8","type":"mqtt in","z":"50b47628.1452b8","name":"","topic":"homebridge/from/get","qos":"2","broker":"8530eb60.3094f8","x":120,"y":1680,"wires":[["78205d54.b463c4"]]},{"id":"78205d54.b463c4","type":"json","z":"50b47628.1452b8","name":"","x":290,"y":1680,"wires":[["c7a91103.987d2"]]},{"id":"c0728aad.612498","type":"debug","z":"50b47628.1452b8","name":"","active":true,"console":"false","complete":"false","x":590,"y":1680,"wires":[]},{"id":"c7a91103.987d2","type":"switch","z":"50b47628.1452b8","name":"","property":"payload.name","propertyType":"msg","rules":[{"t":"eq","v":"flex_lamp","vt":"str"}],"checkall":"true","outputs":1,"x":430,"y":1680,"wires":[["c0728aad.612498","d27ae722.39b008"]]},{"id":"a465611.c7654a","type":"mqtt out","z":"50b47628.1452b8","name":"","topic":"homebridge/to/set","qos":"","retain":"","broker":"45395bbb.44f1c4","x":590,"y":1760,"wires":[]},{"id":"d27ae722.39b008","type":"template","z":"50b47628.1452b8","name":"true","field":"payload","fieldType":"msg","format":"handlebars","syntax":"mustache","template":"{\"name\":\"flex_lamp\",\"characteristic\":\"On\",\"value\":true}","x":390,"y":1760,"wires":[["a465611.c7654a"]]},{"id":"8530eb60.3094f8","type":"mqtt-broker","z":"50b47628.1452b8","broker":"192.168.0.32","port":"1883","clientid":"","usetls":false,"compatmode":true,"keepalive":"60","cleansession":true,"willTopic":"","willQos":"0","willPayload":"","birthTopic":"","birthQos":"0","birthPayload":""},{"id":"45395bbb.44f1c4","type":"mqtt-broker","z":"50b47628.1452b8","broker":"192.168.0.32","port":"1883","clientid":"","usetls":false,"compatmode":true,"keepalive":"60","cleansession":true,"willTopic":"","willQos":"0","willPayload":"","birthTopic":"","birthQos":"0","birthPayload":""}]
```

The example return always `true` so the logic must be expanded and return the value depending on your devise state.

**Comment by danielsen2009** [comment]
I got it figured out! It took me a second to pick through the node red bit, I haven't used it at all in my projects and I wasn't familiar with the other stuff. I though I was missing a lot. I have my devices connect and initiate the mqtt connection and messages themselves. When I send the reply, the "value" is in "quotes". When I tested with mqtt broker without the quotes it started working right away as expected!
